scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -796,7 +796,7 @@ class FossId internal constructor(
                         }
                     }
 
-                    it to snippets.filterNot { it.matchType == MatchType.IGNORED }.toSet()
+                    it to snippets.filterNot { it.matchType in listOf(MatchType.IGNORED, MatchType.NONE) }.toSet()
```

Comment:
Please use `enumSetOf()` and extract it to a variable outside of `filterNot` to not recreate it on each iteration.

Reply:
While at it, you could also do `filterNotTo(mutableSetOf())` instead of `toSet()`.